src/renderer/pages/settings/index.js

```
@@ -82,82 +85,354 @@ let authState = {
+  window.settings.log.info(`initializeTabContent 호출 - 탭 ID: ${tabId}`);
+
+  // 이미 초기화된 탭이면 다시 초기화하지 않음
+  if (tabInitState[tabId]) {
+    window.settings.log.info(`${tabId} 탭은 이미 초기화되어 있습니다.`);
+    return;
+  }
+
+  // 각 탭에 맞는 초기화 함수 호출
+  switch (tabId) {
+    case 'general':
+      initializeGeneralSettings();
```

ESLint

[error] 'initializeGeneralSettings' is not defined.

(no-undef)